packages/curve-ui-kit/src/shared/ui/Metric.tsx

```diff
@@ -20,14 +20,14 @@ const { Spacing, IconSize } = SizesAndSpaces
 export const ALIGNMENTS = ['start', 'center', 'end'] as const
 type Alignment = (typeof ALIGNMENTS)[number]
 
-const MetricSize = {
+export const MetricSize = {
   small: 'highlightM',
   medium: 'highlightL',
   large: 'highlightXl',
   extraLarge: 'highlightXxl',
 } as const satisfies Record<string, TypographyVariantKey>
 
-const MetricUnitSize = {
+export const MetricUnitSize = {
```

Comment:
I know it's used for SymbolCell but exporting this feels smelly to me, these sizes shouldn't be used in other components :rofl: